packages/react-components/src/components/Switch/Switch.tsx

```diff
@@ -42,9 +49,41 @@ export const Switch: React.FC<SwitchProps> = ({
     }
   }, [on]);
 
-  const iconSize: IconSize = size === 'basic' ? 'small' : 'xsmall';
+  const getIcon = (iconSize: IconSize): JSX.Element => {
```

Comment:
It should be converted into a component. Please avoid helpers returning `JSX.Element`.

Reply:
Removed this helper - we have only 2 icons for now and I used simpler approach